scripts/globals/battlefield.lua

```diff
@@ -900,12 +900,16 @@ function Battlefield:onBattlefieldEnter(player, battlefield)
             if type(item) == 'table' then
                 for _, subitem in ipairs(item) do
                     if player:hasKeyItem(subitem) then
-                        player:delKeyItem(subitem)
+                        if (not self.requiredKeyItems.keep) then
```

Comment:
No outer parens here (and below)

Reply:
haha these language switches sometimes get you. thanks!